cloud-tenant-cd/src/main/java/ai/vespa/hosted/cd/cloud/impl/package-info.java

```diff
@@ -0,0 +1,9 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+/**
+ * @author mortent
+ */
+@ExportPackage
```

Comment:
Is this needed?

Reply:
In general, exporting a package is only necessary if another bundle should use your apis programmatically. It's not needed for a component to be constructed.